core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java

```diff
@@ -1351,36 +1364,34 @@ public Set<String> getNamespaceNames() throws ExecutionException {
   }
 
   private void createNamespacesTableIfNotExists(boolean noBackup) throws ExecutionException {
-    if (namespacesTableExists()) {
-      return;
-    }
-
     List<AttributeDefinition> columnsToAttributeDefinitions = new ArrayList<>();
     columnsToAttributeDefinitions.add(
         AttributeDefinition.builder()
             .attributeName(NAMESPACES_ATTR_NAME)
             .attributeType(ScalarAttributeType.S)
             .build());
     try {
-      client.createTable(
-          CreateTableRequest.builder()
-              .attributeDefinitions(columnsToAttributeDefinitions)
-              .keySchema(
-                  KeySchemaElement.builder()
-                      .attributeName(NAMESPACES_ATTR_NAME)
-                      .keyType(KeyType.HASH)
-                      .build())
-              .provisionedThroughput(
-                  ProvisionedThroughput.builder()
-                      .readCapacityUnits(METADATA_TABLES_REQUEST_UNIT)
-                      .writeCapacityUnits(METADATA_TABLES_REQUEST_UNIT)
-                      .build())
-              .tableName(ScalarDbUtils.getFullTableName(metadataNamespace, NAMESPACES_TABLE))
-              .build());
+      if (!namespacesTableExists()) {
+        client.createTable(
+            CreateTableRequest.builder()
+                .attributeDefinitions(columnsToAttributeDefinitions)
+                .keySchema(
+                    KeySchemaElement.builder()
+                        .attributeName(NAMESPACES_ATTR_NAME)
+                        .keyType(KeyType.HASH)
+                        .build())
+                .provisionedThroughput(
+                    ProvisionedThroughput.builder()
+                        .readCapacityUnits(METADATA_TABLES_REQUEST_UNIT)
+                        .writeCapacityUnits(METADATA_TABLES_REQUEST_UNIT)
+                        .build())
+                .tableName(ScalarDbUtils.getFullTableName(metadataNamespace, NAMESPACES_TABLE))
+                .build());
+        waitForTableCreation(Namespace.of(metadataNamespace), NAMESPACES_TABLE);
+      }
```

Comment:
Very minor suggestion. We can move the `columnsToAttributeDefinitions` thing into this if block
```suggestion
      if (!namespacesTableExists()) {
        List<AttributeDefinition> columnsToAttributeDefinitions = new ArrayList<>();
        columnsToAttributeDefinitions.add(
            AttributeDefinition.builder()
                .attributeName(NAMESPACES_ATTR_NAME)
                .attributeType(ScalarAttributeType.S)
                .build());
        client.createTable(
            CreateTableRequest.builder()
                .attributeDefinitions(columnsToAttributeDefinitions)
                .keySchema(
                    KeySchemaElement.builder()
                        .attributeName(NAMESPACES_ATTR_NAME)
                        .keyType(KeyType.HASH)
                        .build())
                .provisionedThroughput(
                    ProvisionedThroughput.builder()
                        .readCapacityUnits(METADATA_TABLES_REQUEST_UNIT)
                        .writeCapacityUnits(METADATA_TABLES_REQUEST_UNIT)
                        .build())
                .tableName(ScalarDbUtils.getFullTableName(metadataNamespace, NAMESPACES_TABLE))
                .build());
        waitForTableCreation(Namespace.of(metadataNamespace), NAMESPACES_TABLE);
      }
```

Reply:
Yes, thank you!
https://github.com/scalar-labs/scalardb/pull/1107/commits/6cac45bc96046515b66cc2778f3158ddac1a8f4e